CHANGELOG.md

```diff
@@ -55,6 +55,13 @@
 
 #### Under the hood
 
+- New `v1` versions of `CustomResourceDefinitions` introduced for KIC 2.0 are now
+  backwards compatible with the previous `v1beta1` CRD definitions. In practice
```

Comment:
"backwards compatability" do you men "migrate from v1beta1 to v1" ? Or support both version ?
Kindly add some clarity.

Reply:
backwards compatibility, meaning v1 is now compatible backwards [with v1beta1]

Reply:
how do user expect from the "backwards compatibility" ? do you mind provide more clear/concrete statement, both v1 & v1beta1, from v1beta1 -> v1, from v1-beta1 ?

Reply:
Understood, let me add a small diagram to that effect, let me know if you feel this helps:
```suggestion
  backwards compatible with the previous `v1beta1` CRD definitions (i.e. `v1beta1 -> v1`
  upgrades of KIC's CustomResourceDefinitions now work fully automatically). In practice
```